src/photos.cs

```diff
@@ -1005,9 +1005,15 @@ interface PHPhotoLibrary {
 
 	[iOS (9,1)]
 	[TV (10,0)]
+#if MONOMAC
+	[DisableDefaultCtor] // NS_UNAVAILABLE
```

Comment:
Ohh. I'm silly. So in 10.12 / Xcode 8b1+ they added a bunch of Copying  / Encoding bits that I added.

However, in older OS X, we don't have them and the tests fail (since they are run there).

I'll teach the test to skip those before 10.12 (and commit here).